modules/admin-ui/src/main/webapp/index.html

```diff
@@ -318,7 +316,6 @@
     <script src="scripts/shared/controllers/navigationController.js"></script>
     <script src="scripts/shared/controllers/applicationController.js"></script>
     <script src="scripts/shared/controllers/applicationController.js"></script>
-    <script src="scripts/shared/controllers/userPreferencesController.js"></script>
```

Comment:
applicationController is a duplicate, mind kicking one of them out?

Reply:
Done.